lxd/instance/drivers/driver_qemu.go

```diff
@@ -504,6 +506,12 @@ func (vm *qemu) Freeze() error {
 
 // onStop is run when the instance stops.
 func (vm *qemu) onStop(target string) error {
+	ctxMap := log.Ctx{
```

Comment:
@stgraber not a blocker now but would be good to add a common logger property like we do for networks, devices and storage in the future.